chaos.c

```diff
@@ -9,7 +9,7 @@
 #include <unistd.h>
 #include <string.h>
 #include <sys/un.h>
-#ifdef __HAIKU__
+#if defined(__HAIKU__) || defined(__OpenBSD__)
```

Comment:
I appreciate leaving old code alone, but in this case I think we should go out on a limb and always use <errno.h>.

Reply:
I will do that later today when I push an update.